specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2024-08-01-preview/cluster.json

```diff
@@ -2004,9 +2004,31 @@
           "type": "string",
           "x-ms-client-name": "password",
           "description": "The gateway settings user password."
+        },
+        "restAuthMsEntra.users": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "objectId": {
+                "type": "string",
+                "description": "The ObjectID of the Entra user or client ID of the enterprise applications."
+              },
+              "displayName": {
+                "type": "string",
+                "description": "The display name of the Entra user."
+              },
+              "UPN": {
+                "type": "string",
+                "description": "The User Principal Name(UPN) of the Entra user. It may be empty in certain cases, such as for enterprise applications"
```

Comment:
>Name(UPN) [](http://example.com/codeflow?start=51&length=9)

missing space

Reply:
Resolved in latest commit, Please check commit ID a485479.